test/go.mod

```diff
@@ -2,9 +2,9 @@ module sigs.k8s.io/cluster-api-provider-vsphere/test
 
 go 1.22.0
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.8.1
+replace sigs.k8s.io/cluster-api => github.com/chrischdi/cluster-api v1.8.0-rc.0.0.20240823101941-995201a28568
```

Comment:
When we bump to a commint on release-1.8 later. Let's please bump top-level & test/go.mod